LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) Maho Contributors
```

Comment:
I don't know about this, to me this "contributors" thing (which doesn't really represent anybody) creates possible problems in the future

Reply:
It’s what the Linux Foundation recommends, except I know the word “The” bothered you so I removed it.

https://www.linuxfoundation.org/blog/blog/copyright-notices-in-open-source-software-projects

It makes sense to me because even if Maho is a legal entity, there’s no CLA required to be signed so copyright remains with the “contributors.”

But I didn’t include that word intentionally to rehash any sort of discussion, and I’m really okay with whatever.

Reply:
At least all projects in the organization should have the same copyright. I truly don't understand in practical what giving copyright to "xxx contributors" legally means. I would even remove the file and just mark the repository as MIT in composer.json and repository settings.

Reply:
Really the copyright notice has no legal meaning, as you know all contributors retain their copyright anyway. Just as long as other projects don't remove the notice if they fork, etc.

It's why some programmers write absurd things as:
```
Copyright © 2024. Written by a team of cats walking across keyboards.
Copyright © 2024. All bytes reserved.
Copyright © 2024. All rights reserved. Some wrongs are still available.
```

So in effect the notice doesn't actually assign the copyright to anyone, it just acknowledges the copyright belongs to individual contributors to a particular project.

But in any case, I'd keep the LICENSE.txt for two reasons:

1) I don't think you can specify the license on a GitHub repo without one.
2) Without the LICENSE.txt, any other project can fork and remove all references to Maho.

Reply:
> https://github.com/laravel/laravel for example doesn't have a license file, only mentions the license in composerjson

Their justification is:

> The reason there is not a license in this repository is that this is the project skeleton used to make other applications. Taylor shouldn't automatically own everyone's applications.
> https://github.com/laravel/laravel/pull/4903#issuecomment-453785517

That doesn't really apply here though. Also they don't get the license tab in their repo:

![image](https://github.com/user-attachments/assets/4eb5c4c5-7ac3-4253-acb5-3e061ba6ede8)

Reply:
But if "maho" doesn't exist, then "maho contributors" is not a legal entity either. I think that an uncontrollable (because once it's set it's set) copyright holder/owner can be a problem (see the GPL modules that are stuck with GPL because "how can we ask everybody" and so all that code is thrown away because it's unusable if you want to follow the licenses correctly as everybody should). and if people write "copyright by cats" then I think we can have the project name there, in my opinion.

Reply:
The maho contributors are you, me, and others, and we definitely exist! It would be similar to something like "the xxx volunteers" or whatever even if "xxx" isn't a legal entity. The point about "copyright by cats" was exactly that the word doesn't change anything legally, but I understand your point about being stuck with it forever, so ultimately it's down to preference.

Anyway, I sincerely didn't mean for this to be such a sticking point, I simply thought it was the word "The" and the year you objected to. Feel free to change it, else I can do tomorrow.